cmake/thirdparty/get_libcudacxx.cmake

```diff
@@ -13,22 +13,12 @@
 # =============================================================================
 
 # Use CPM to find or clone thrust
-function(find_and_configure_libcudacxx VERSION)
-    rapids_cpm_find(
-        libcudacxx         ${VERSION}
-        CPM_ARGS
-            GIT_REPOSITORY https://github.com/NVIDIA/libcudacxx.git
-            GIT_TAG        ${VERSION}
-            GIT_SHALLOW    TRUE
-            DOWNLOAD_ONLY  TRUE
-    )
-    # TODO: Once libcu++ exports a target, use that instead
-    add_library(libcudacxx INTERFACE)
-    target_include_directories(libcudacxx
-        INTERFACE   $<BUILD_INTERFACE:${libcudacxx_SOURCE_DIR}/include>
-                    $<INSTALL_INTERFACE:include/cuco/libcudacxx>)
-    install(DIRECTORY ${libcudacxx_SOURCE_DIR}/include/ DESTINATION include/cuco/libcudacxx)
-    install(DIRECTORY ${libcudacxx_SOURCE_DIR}/libcxx/include/ DESTINATION include/cuco/libcxx/include)
+function(find_and_configure_libcudacxx)
+    include(${rapids-cmake-dir}/cpm/libcudacxx.cmake)
+
+    rapids_cpm_libcudacxx(BUILD_EXPORT_SET cuco-exports
+                          INSTALL_EXPORT_SET cuco-exports)
+
 endfunction()
 
-find_and_configure_libcudacxx(1.4.0)
+find_and_configure_libcudacxx()
```

Comment:
How do we specify a version or git hash now?

Reply:
https://github.com/NVIDIA/cuCollections/pull/107/files#diff-e31973fefe5cbec158e22ade93d7f06821cc81f670ba932178c8de68d433da0eR15-R33

Talked with @robertmaynard , and here's the solution we have for now.